setup.py

```diff
@@ -47,7 +47,7 @@
         'setuptools',
         'z3c.jbot',
         'plone.directives.form',
-        'collective.z3cform.datagridfield > 1.1',
+        'collective.z3cform.datagridfield >= 1.1',
```

Comment:
Currently the latest officially released version is indeed 1.1, so this seems fine. Do the tests pass? Though maybe it depends on unreleased version of `collective.z3cform.datagridfield`. In that case it would be good to make a release so we can depend on it. Please discuss it with the add-on author.